guava-testlib/src/module-info.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2024 The Guava Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Guava Testlib
+ */
+open module com.google.common.testlib {
```

Comment:
It seems fine to be more permissive for the test library here by using `open module`: The stakes are just lower for `guava-testlib` than for `guava`. So, at this point, I'm just curious as to whether this ends up being necessary for something in particular or whether you left it this way to be conservative in general. I think that _our_ tests pass even without `open`, assuming that I tested it right.

Reply:
Left it this way to be conservative in general :) although happy to drop as well.